coriolis/conductor/rpc/server.py

```diff
@@ -1029,14 +1029,18 @@ def execute_replica_tasks(self, ctxt, replica_id, shutdown_instances):
 
     @replica_synchronized
     def get_replica_tasks_executions(self, ctxt, replica_id,
-                                     include_tasks=False):
+                                     include_tasks=False, include_info=False,
+                                     to_dict=True):
```

Comment:
Is there any reason for adding the `to_dict` kwarg here and not using it anywhere else?

If it's a pickling issue and we can't send a DB object over oslo_messaging then we should always set this to `True` at the DB layer IMO and never propagate the non-dict objects down the stack.